android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -257,56 +259,80 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
       }
     });
 
-    return webView;
+    return wrapper;
   }
 
   @ReactProp(name = "javaScriptEnabled")
-  public void setJavaScriptEnabled(WebView view, boolean enabled) {
-    view.getSettings().setJavaScriptEnabled(enabled);
+  public void setJavaScriptEnabled(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setJavaScriptEnabled(enabled);
   }
 
   @ReactProp(name = "setBuiltInZoomControls")
-  public void setBuiltInZoomControls(WebView view, boolean enabled) {
-    view.getSettings().setBuiltInZoomControls(enabled);
+  public void setBuiltInZoomControls(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setBuiltInZoomControls(enabled);
   }
 
   @ReactProp(name = "setDisplayZoomControls")
-  public void setDisplayZoomControls(WebView view, boolean enabled) {
-    view.getSettings().setDisplayZoomControls(enabled);
+  public void setDisplayZoomControls(RNCWebView view, boolean enabled) {
+    view.getWebView().getSettings().setDisplayZoomControls(enabled);
   }
 
   @ReactProp(name = "setSupportMultipleWindows")
-  public void setSupportMultipleWindows(WebView view, boolean enabled){
-    view.getSettings().setSupportMultipleWindows(enabled);
+  public void setSupportMultipleWindows(RNCWebView view, boolean enabled){
+    view.getWebView().getSettings().setSupportMultipleWindows(enabled);
+  }
+
+  @ReactProp(name = "webViewKey")
+  public void setWebViewKey(RNCWebView view, String webViewKey) {
```

Comment:
unlike iOS, where there is a `didMoveToWindow` function, if there is a webViewKey, we rettach the view at this point